lca_website/ngo/models.py

```diff
@@ -1,3 +1,26 @@
 from django.db import models
 
-# Create your models here.
+
+class ngo(models.Model):
+    """
+    A model to represent an NGO. The Registration number of the NGO is used as the primary key in the database. 
+    Since the category is chosen from a set of categories, it is made into an enum and stored as an integer.
+    The website is made an optional field.   
+    
+    """
+
+    TYPE_ZERO_NGO = 0
+    TYPE_ONE_NGO = 1
+
+    NGO_CATEGORY = (
```

Comment:
I feel defining it multiple times might be redundant and lead to inconsistency. Can this be made a global thing?

Reply:
Defined it in models globally and imported it wherever needed. Is it okay?